.travis.yml

```diff
@@ -80,6 +80,9 @@ jobs:
           && npm init -y && npm install --save-dev @commitlint/travis-cli && popd
         # yamllint enable rule:line-length
         - export PATH=/opt/commitlint/node_modules/.bin:$PATH
+        # install pylint and prettytable.
+        - sudo apt-get install python3-prettytable -y
+        - sudo apt install pylint3
```

Comment:
installing the packages with the same command is normally a little faster. If that does not work for some reason, there needs to be a comment about it